spec/index.html

```diff
@@ -591,16 +588,12 @@ <h4>Terminology</h4>
               <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank node</a>
             </li>
           </ul>
-          <p>In addition, we define the following terms:</p>
+          <p>In addition, we define the following term:</p>
           <ul>
             <li>
               <a class="type" href="#defn_RDFTerm">RDF Term</a>, which includes IRIs, blank nodes and
               literals
```

Comment:
I suppose quoted triples will be added later, when SPARQL 1.2 is brought in line with RDF 1.2 (vs the current change set which seems to bring SPARQL 1.1 in line with RDF 1.1)?

Reply:
Yep, that's the plan.